<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="HomePage Test"
           tests="2"
           failures="2"
           errors="0"
           skipped="0"
           time="0.490">
  <properties />
            <testcase name="testCase" time="0.195" classname="testCase.hiBug">
                <failure type="net.sf.json.JSONException"
                  message="null object"
              >
<![CDATA[net.sf.json.JSONException: null object
  net.sf.json.JSONObject.verifyIsNull(JSONObject.java:2744)
  net.sf.json.JSONObject.get(JSONObject.java:1957)
  testCase.hiBug.testCase(hiBug.java:49)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
  org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>

          </testcase>
            <testcase name="editBug" time="0.295" classname="testCase.hiBug">
                <failure type="net.sf.json.JSONException"
                  message="null object"
              >
<![CDATA[net.sf.json.JSONException: null object
  net.sf.json.JSONObject.verifyIsNull(JSONObject.java:2744)
  net.sf.json.JSONObject.get(JSONObject.java:1957)
  testCase.hiBug.editBug(hiBug.java:77)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
  org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
